docs/pages/_app.js

```diff
@@ -220,7 +220,7 @@ function AppWrapper(props) {
           { text: `v${materialPkgJson.version}`, current: true },
           {
             text: 'View all versions',
-            href: `https://mui.com${languagePrefix}/versions/`,
+            href: `https://mui.com${languagePrefix}/material-ui/getting-started/versions/`,
```

Comment:
Q: when is `productId === 'core'` true?

Reply:
I beloeve the `=== 'core'` verification is there just so we can show in which category each product is (either Core or X). I'm in the process of removing external mentions to "Core" as that should be just an internal way to name our team—it's not meaningful for the product and has been too confusing 😅